learning-another-language/hello-world.md

```diff
@@ -0,0 +1,279 @@
+
+The following steps will let us create and run JavaScript programs with ease:
+
+1. Create an account on [replit.com](https://replit.com/) and log-in
+1. Visit [the Hello, World! REPL](https://replit.com/@adacore/hello-world#index.js)
+
+On the right side, we see the _console_. We can use this console to execute small, quick JavaScript snippets, such as `2 + 3`, `4 * 5`, or any other valid code.
+
+![JavaScript REPL. In the console on the right-side, the line "2 + 3" is executed, and shows a result of "5." The line "4 * 5" is executed, and shows a result of "20."](../assets/learning-a-new-language_hello-world_repl-console-demo.png)
+
+On the left side, we see a file explorer, and `index.js`. We can write JavaScript code within this file.
+
+Then, we can run this code using the green "Run" button on the top.
+
+![JavaScript REPL. The green "Run" button on the top is highlighted.](../assets/learning-a-new-language_hello-world_repl-run-demo.png)
+
+There are more features in replit.com that are worth exploring! For now, let's begin working on our Hello, World program.
+
+## Implementation
+
+To print `Hello, World!` in JavaScript, [we can write](https://replit.com/@adacore/hello-world#index.js):
+
+<!-- prettier-ignore-start -->
+```js
+console.log('Hello, World!');
+```
+<!-- prettier-ignore-end -->
+
+[When we run our code](https://replit.com/@adacore/hello-world#index.js), we should see `Hello, World!` in the console.
+
+| <div style="min-width:200px;"> Piece of Code </div> | Notes                                                                                                             |
+| --------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `console`                                           | A variable. The word "console" usually refers to the Terminal, command line, or wherever output logs get printed. |
+| `console.log( ... )`                                | A function call that takes in some arguments.                                                                    |
+| `'Hello, World!'`                                   | A string literal. This string is surrounded by single-quotes (`'`).                                               |
+| `;`                                                 | A semicolon.                                                                                                      |
+
+Let's debrief in slightly more detail.
+
+### The Console
+
+`console` seems a little random here. It's lowercase, like Python variables, but there seems to be no variable assignment. We don't see any `import`s or anything like that, so where did `console` come from?
+
+`console` is a variable that references a specific object: the console! This console object gives us access to wherever the program prints out output.
+
+It seems that `console` is within scope of our JavaScript file by default.
```

Comment:
Be a little more transparent in the thinking that leads to the conclusion of this being something in scope. The existing line sounds like the author isn't quite sure to me.
```suggestion
Since we were able to use it without any setup, we might guess that `console` is in scope in our JavaScript file by default.
```